File: src/test/Top_Level_Test.java
package test;


import java.util.ArrayList;
import java.util.List;

import top_level.JoinPropKt;
import top_level.StringFunctions;

/**
 * Created by zhaoyu on 2017/5/31.
 */
public class Top_Level_Test {

    public static void main(String... a) {
        List<String> items = new ArrayList<>();
        items.add("one");
        items.add("two");
        items.add("three");
        String str = StringFunctions.joinToString(items, ", ", "(", ")");

        System.out.println(str);

        JoinPropKt.getOpCount();

        System.out.println("====");

        System.out.println(StringFunctions.lastChar("better"));

    }
}

File: src/test/Top_Level_Test.kt
package test
import top_level.join
import top_level.joinString
import top_level.lastChar as last       // 导入

/**
 * Created by zhaoyu on 2017/6/1.
 */
fun main(args: Array<String>) {
    println("Kotlin".last())

    println("========")

    val list = arrayListOf("1","2","3")
    println(list.joinString(","))
    println(list.join(","))
}